### ISSUE TITLE
Needs update to remove the warning "ESLINT_LEGACY_OBJECT_REST_SPREAD"

### ISSUE THREAD

**Issue by macabeus** [pagarme/react-style-guide#18]
When I use this lint, I receive this warning:

```
(node:36288) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-airbnb/node_modules/eslint-config-airbnb-base/index.js")
```

It's happens because the version of the package [`eslint-config-airbnb`](https://github.com/pagarme/react-style-guide/blob/14fe66dc2ce65b501944a48ff595378e7a49bd1e/packages/eslint/package.json#L26) is outdated, then it needs be upgrade to version `17.1.0` in order to resolve it.
